CHANGES.rst

```diff
@@ -6,6 +6,8 @@ Unreleased
 -   Use modern packaging metadata with ``pyproject.toml`` instead of ``setup.cfg``.
     :pr:`4947`
 -   Ensure subdomains are applied with nested blueprints. :issue:`4834`
+-   Automatically setup project and install dependencies with the
```

Comment:
This doesn't need a change entry, as it's not a code change that affects users installing the library.